ruby/src/main/java/com/google/protobuf/jruby/RubyDescriptor.java

```diff
@@ -215,8 +215,8 @@ public IRubyObject allocate(Ruby runtime, RubyClass klazz) {
     klass.include(new IRubyObject[] {messageExts});
     klass.instance_variable_set(runtime.newString(Utils.DESCRIPTOR_INSTANCE_VAR), this);
     klass.defineAnnotatedMethods(RubyMessage.class);
-    // Workaround for https://github.com/jruby/jruby/issues/7154
-    klass.searchMethod("respond_to?").setIsBuiltin(false);
+        // Workaround for https://github.com/jruby/jruby/issues/7154
```

Comment:
It isn't clear that this indentation change is intended.

Reply:
Reverted.